api/pkg/apis/v1alpha1/managers/secrets/ReadMe.md

```diff
@@ -0,0 +1,11 @@
+# Secret Manager
+
+The Secret Manager implements the `IExtSecretProvider` interface and is part of the `EvaluationContext`, which is used in expression parsing. It only has one function, which is reading the secret object, field field with a local evaluation context if provided.
+
+The Secret Manager decides which secret provider to use when processing the parsing request. It has three options:
+
+1. **Secret Provider Name Specified**: If the secret provider name is specified like `Get("k8s::sampleobject", "samplefield", nil)`, then that specific secret provider is used.
```

Comment:
How to determine the k8s namespace?

Reply:
The NS is determined by the evaluation context. Will update the doc with the logging change.